### ISSUE TITLE
submission blocked on Argoverse 2

### ISSUE THREAD

**Issue by kingwmk** [Cloud-CV/EvalAI#4340]
Hello. I have a submission blocked problem on Argoverse 2: Motion Forecasting Competition's Multi-world prediction channel. Thank you for your help. https://eval.ai/web/challenges/challenge-page/1719/my-submission

**Comment by kingwmk** [comment]
The status always is 'submitted'.

**Comment by RishabhJain2018** [comment]
Thanks @kingwmk for reaching out.

@gchhablani Can you please look into it?

**Comment by lon0862** [comment]
I have the same question, I have a submission blocked problem on Argoverse 2: Motion Forecasting Competition's single agent forecasting. It always in "submitted" status.

**Comment by kingwmk** [comment]
Thank you so much for fixing this issue. However, I get a new problem that I can not upload the submission.parquet file. It blocked in the uploading process @gchhablani 
![image](https://github.com/Cloud-CV/EvalAI/assets/19169383/e73b06ce-b366-453f-9c8f-6e6bed91c127)

**Comment by kingwmk** [comment]
Sorry. The problem come out again. @RishabhJain2018 @gchhablani

**Comment by RishabhJain2018** [comment]
Hey Everyone, Closing this since this isn't the issue anymore. Please feel free to open a new issue if it persists.
